sass/layout/_grid.scss

```diff
@@ -0,0 +1,9 @@
+@use '../abstracts' as *;
+
+main{
+    max-width: 140rem;
+    margin: 0 auto;
+}
+// body{
```

Comment:
Avoid commenting on old code, it is always better to remove old code instead of commenting out. In real-world projects, it can cause confusion and can clutter the code if other devs start following the same practice